src/openforms/js/components/admin/form_design/variables/ServiceFetchConfigurationPicker.js

```diff
@@ -0,0 +1,94 @@
+import {useFormik} from 'formik';
+import PropTypes from 'prop-types';
+import React, {useContext, useState} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import Field from 'components/admin/forms/Field';
+import FormRow from 'components/admin/forms/FormRow';
+import Select from 'components/admin/forms/Select';
+
+import {FormLogicContext} from './../Context';
+import ServiceFetchConfigurationForm from './ServiceFetchConfigurationForm';
+
+const ServiceFetchConfigurationPicker = ({data = {}, onChange, onFormSave}) => {
+  const formLogicContext = useContext(FormLogicContext);
+
+  const [selectExisting, setSelectExisting] = useState(false);
+  const [selectedServiceFetchConfig, setSelectedServiceFetchConfig] = useState(null);
+
+  const serviceFetchConfigurationChoices = formLogicContext.serviceFetchConfigurations.map(
+    fetchConfig => {
+      return [
+        fetchConfig.url,
+        fetchConfig.name || `${fetchConfig.method} ${fetchConfig.path} (${fetchConfig.service})`,
+      ];
+    }
+  );
+
+  const formik = useFormik({
+    initialValues: {
+      method: 'GET',
+      service: '',
+      path: '',
+      queryParams: [['', ['']]],
+      headers: [['', '']],
+      body: '',
+      dataMappingType: '',
+      mappingExpression: '',
+    },
+    onSubmit: (values, {setSubmitting}) => {
+      alert(JSON.stringify(values, null, 2));
+    },
+  });
+
+  return (
+    <div className="servicefetchconfiguration-picker">
+      <div className="servicefetchconfiguration-select">
+        <FormRow>
+          <Field
+            name={'fetchConfiguration.existing'}
+            label={
+              <FormattedMessage
+                defaultMessage="Choose existing configuration"
+                description="Service fetch configuration modal select existing field label"
+              />
+            }
+          >
+            <Select
+              name="fetchConfiguration.existing"
```

Comment:
this *shouldn't* be necessary to repeat because the parent `Field` should inject that, but maybe Formik requires it?

Reply:
This field isn't actually managed by Formik, so I think I can just remove the duplication